cylc/flow/cfgspec/globalcfg.py

```diff
@@ -101,6 +101,43 @@
              desc='''
             The number of old run directory trees to retain at start-up.
         ''')
+        with Conf('run hosts', desc='''
+        Configure allowed suite hosts and ports for starting up (running or
+        restarting) suites. Additionally configure host selection settings
+        specifying how to determine the most suitable run host at any given
+        time from those configured.
+    '''):
+            Conf('available', VDR.V_SPACELESS_STRING_LIST, desc='''
+                A list of allowed suite run hosts. One of these hosts will be
+                appointed for a suite to start up on if an explicit host is not
+                provided as an option to a ``run`` or ``restart`` command.
+            ''')
+            Conf(
+                'ports', VDR.V_INTEGER_LIST, list(range(43001, 43101)),
+                desc='''
+                    A list of allowed ports for Cylc to use to run suites.
+            ''')
+            Conf('condemned', VDR.V_ABSOLUTE_HOST_LIST, desc='''
+                Hosts specified in ``condemned hosts`` will not be considered
+                as suite run hosts. If suites are already running on
+                ``condemned hosts`` they will be automatically shutdown and
+                restarted (see:ref:`auto-stop-restart`).
+            ''')
+            Conf('auto restart delay', VDR.V_INTERVAL, desc='''
+                Relates to Cylc's auto stop-restart mechanism (see
+                :ref:`auto-stop-restart`).  When a host is set to automatically
+                shutdown/restart it will first wait a random period of time
+                between zero and ``auto restart delay`` seconds before
+                beginning the process. This is to prevent large numbers of
+                suites from restarting simultaneously.
+            ''')
```

Comment:
Sorry, missed this in my first review - according to the proposal, this is the only one meant to be directly under `scheduler`, not `run hosts`